apps/help-center/help-center-gutenberg-disconnected.js

```diff
@@ -22,7 +22,7 @@ function HelpCenterContent() {
 				href="https://wordpress.com/help"
 				icon={ <HelpIcon /> }
 				label="Help"
-				size={ canvasMode === 'edit' ? 'compact' : undefined }
+				size={ ! canvasMode || canvasMode === 'edit' ? 'compact' : undefined }
```

Comment:
I guess this would be the same as

```suggestion
				size={ canvasMode !== 'edit' ?  undefined : 'compact' }
```

Reply:
No, in this case we need falsy `canvasMode` to result in `compact` being applied. Otherwise in the post editor (where there currently isn't a canvas mode) the icon will be displayed at the full size instead of compact.